networking/p2p/src/main/java/tech/pegasys/artemis/networking/p2p/MothraP2PNetwork.java

```diff
@@ -61,23 +64,34 @@ public MothraP2PNetwork(
       String networkInterface,
       String identity,
       String bootnodes,
-      boolean isBootnode) {
+      boolean isBootnode,
+      List<String> peers) {
     this.eventBus = eventBus;
     this.store = store;
     this.port = port;
     this.networkInterface = networkInterface;
     this.identity = identity;
     this.bootnodes = bootnodes;
     this.isBootnode = isBootnode;
+    this.peers = peers;
     this.gossipProtocol = GossipProtocol.GOSSIPSUB;
     eventBus.register(this);
     this.handler = new MothraHandler(this.eventBus, this.store);
     mothra.Init();
     mothra.DiscoveryMessage = this.handler::handleDiscoveryMessage;
     mothra.ReceivedGossipMessage = this.handler::handleGossipMessage;
     mothra.ReceivedRPCMessage = this.handler::handleRPCMessage;
+    if (this.peers.size() == 0) {
```

Comment:
List#isEmpty is a little safer for the interface type, as it may avoid counting size to determine emptiness.

Reply:
agreed